File: pbwebbowcl/src/app/console-applicativa/commons/dto/MonitoraggioServiziDTO.ts
/*
 * Copyright Regione Piemonte - 2024 
 * SPDX-License-Identifier: EUPL-1.2 
*/


export class ConsoleMonitoraggioServiziDTO{

  constructor(
   public esito: string,
   public  codiceErrore: string,
	 public messaggioErrore : string,
	 public dataInizioChiamata: Date,
   public  dataFineChiamata:Date,
	 public nomeServizio: string,
   public categoriaServizio: string
  ){

  }


  public static createConsoleMonitoraggioServiziDTO(): ConsoleMonitoraggioServiziDTO{
    var filter = new ConsoleMonitoraggioServiziDTO(undefined, undefined, undefined, undefined, undefined, undefined,undefined)
    return filter

}
 }

File: pbwebbo/pbwebbo-jar/src/main/java/it/csi/pbandi/pbwebbo/dto/MonitoraggioServiziDTO.java
/*******************************************************************************
* Copyright Regione Piemonte - 2024
* SPDX-License-Identifier: EUPL-1.2
******************************************************************************/

package it.csi.pbandi.pbwebbo.dto;

import java.util.Date;

public class MonitoraggioServiziDTO {
	public String esito;
	public String codiceErrore;
	public String messaggioErrore;
	public Date dataInizioChiamata;
	public Date dataFineChiamata;
	public String nomeServizio;
	public String categoriaServizio;
	
	public String getEsito() {
		return esito;
	}
	public void setEsito(String esito) {
		this.esito = esito;
	}
	public String getCodiceErrore() {
		return codiceErrore;
	}
	public void setCodiceErrore(String codiceErrore) {
		this.codiceErrore = codiceErrore;
	}
	public String getMessaggioErrore() {
		return messaggioErrore;
	}
	public void setMessaggioErrore(String messaggioErrore) {
		this.messaggioErrore = messaggioErrore;
	}
	public Date getDataInizioChiamata() {
		return dataInizioChiamata;
	}
	public void setDataInizioChiamata(Date dataInizioChiamata) {
		this.dataInizioChiamata = dataInizioChiamata;
	}
	public Date getDataFineChiamata() {
		return dataFineChiamata;
	}
	public void setDataFineChiamata(Date dataFineChiamata) {
		this.dataFineChiamata = dataFineChiamata;
	}
	public String getNomeServizio() {
		return nomeServizio;
	}
	public void setNomeServizio(String nomeServizio) {
		this.nomeServizio = nomeServizio;
	}
	public String getCategoriaServizio() {
		return categoriaServizio;
	}
	public void setCategoriaServizio(String categoriaServizio) {
		this.categoriaServizio = categoriaServizio;
	}

	
	
	
	

}
